autoload/phpactor.vim

```diff
@@ -435,14 +435,41 @@ endfunction
 " RPC -->-->-->-->-->--
 """""""""""""""""""""""
 
+let g:phpactorProjectRootPatterns = ['composer.json', '.git']
+let g:phpactorGlobalRootPatterns = ['/', '/home']
```

Comment:
can we use `$HOME` somehow? if the user had `.git` in their home directory Phpactor could potentially do some crazy things.